ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -122,3 +140,17 @@ function RJSFForm(props) {
     </MuiThemeProvider>
   )
 }
+
+const LoadingComponent = () => (
+  <div style={{
```

Comment:
@Abhishek-kumar09 can the contents of this LoadingComponent be defined by the extension, instead of being defined here? In MeshMap, we would like to use the custom loading spinner instead of the `CircularProgress` component.